### ISSUE TITLE
Not working in Germany

### ISSUE THREAD

**Issue by GermanGuy1986** [NeApp/neon-extension#31]
Hey,
i'm from germany and using netflix.de and amazon.de.

Seems as if Radon is not working here, am i right?

Thanks

**Comment by fuzeman** [comment]
It looks like some changes will be required to support additional regions for Amazon Video, currently only https://amazon.com is supported. I'll look at adding support for more regions in a future release.

----------

Not sure why Netflix isn't working for you, it should support all regions from my tests. To help identify this issue, could you post the logger messages by doing the following steps:

1. Enable "Trace" logging in Radon *(Configuration -> Debugging -> Logging -> Log Level)*
2. Visit Netflix (or refresh existing tab)
3. Open the console *(Chrome: Ctrl + Shift + J, Firefox: Ctrl + Shift + K)*
4. Start a video *(watch for at least a minute)*
5. Check the console for messages, screenshot or paste them here.

--------

You should see messages like the following in the console:
```
Bound to "neon.request"
Initializing "activity" service "neon-extension-source-netflix:activity"...
Initialized "activity" service "neon-extension-source-netflix:activity"
```
```
Navigated to  /watch/... (from  /browse )
```
```
Fetching metadata for "..." (item: ... )
Created session: Object { ... }
```

**Comment by GermanGuy1986** [comment]
![image](https://user-images.githubusercontent.com/42896711/45020751-1ca08200-b030-11e8-9665-142eb30feb99.png)

**Comment by fuzeman** [comment]
Sorry, you also need to click on the drop-down in the console that says "Default levels" (visible in your screenshot) and enable "Verbose" logging.

I think this might be caused by the episode identifier format (`St. 1: Flg. 3` in your screenshot), the logger messages should be able to confirm this. You could also confirm this is the case by changing Netflix to English to see if it works.

**Comment by fuzeman** [comment]
Support for localized episode identifiers in Netflix has been included in [v2.2.0-beta.3](https://github.com/RadonApp/radon-extension/releases/tag/v2.2.0-beta.3).

**Note:** Shows with localized titles aren't supported yet, this will be available with the addition of mappings in a future update.
